examples/hello-world/.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+  // Use IntelliSense to learn about possible Node.js debug attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch Program",
+      "program": "${workspaceFolder}/index.js"
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Run Mocha tests",
+      "program": "${workspaceRoot}/node_modules/.bin/_mocha",
+      "cwd": "${workspaceRoot}",
+      "args": [
+        "--opts",
+        "${workspaceRoot}/test/mocha.opts",
+        "dist*/test/**/*.js",
```

Comment:
This will not work. `npm run build` produces both `dist8` and `dist10` directories. Your wildcard will match both Node.js 8.x and 10.x dist files, therefore each test will be run twice, and if you run on Node.js 8.x, the 10.x tests will fail on unsupported syntax/APIs.